<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>35</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
    background-color:#fdf1d9;
    font-family:Poiret One;
    font: bold;
}

QWidget#Form{
	background-color:#ffc05c;
	border-style: solid;
    border-width: 1px;
    border-radius: 5px;
    border-color: #ffc05c;
}

QPushButton{
    background-color: #ffc05c;
    border-style: solid;
    border-width: 2px;
    border-radius: 12px;
    border-color: #ffc05c;
    font: bold 18px;
    padding: 8px;
	color:#2584a7;

}

QPushButton:checked{
	border-color: #fe6d73;
}

QPushButton:hover{
	border-color: #fe6d73;
	/*border-width: 3px;
	border-radius: 8px;*/
}

QPushButton:pressed{
    border-width: 3px;
	border-radius: 10px;
	border-color: #fe6d73;
}

QFrame#frame{
	background-color:#fdf1d9;
 	border-style: solid;
    border-width: 3px;
    border-radius: 17px;
    border-color: #ffc05c;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item row="0" column="2">
       <widget class="QPushButton" name="pB_reset">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Remplacer</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icon_reset.png</normaloff>icon_reset.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Recette</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
